modules/utils/src/merkle.ts

```diff
@@ -15,7 +16,9 @@ export const generateMerkleTreeData = (transfers: CoreTransferState[]): { root:
   });
 
   // Generate tree
-  const tree = new MerkleTree(leaves, keccak256, { sortPairs: true });
+  const tree = new merkle.Tree();
+  tree.insert_hex_js(leaves);
```

Comment:
The Rust implementation does it's own sorting. So, you can get rid of the sorting by `transferId` above.